addons/api/mirage/serializers/role.js

```diff
@@ -18,7 +18,19 @@ export default ApplicationSerializer.extend({
       id: group.id,
       type: 'group',
     }));
-    json.principals = [...serializedUsers, ...serializedGroups];
+
+    const serializedManagedGroups = model.managedGroups.models.map(
+      (managedGroup) => ({
+        scope_id: model.scope.id,
+        id: managedGroup.id,
+        type: 'managed group',
```

Comment:
```suggestion
        type: 'managed-group',
```

Reply:
the type is `managed group` 

<img width="404" alt="Screen Shot 2022-12-16 at 12 06 40 PM" src="https://user-images.githubusercontent.com/15043878/208180125-12dc3805-e082-4cc8-8548-1c9b356b9486.png">

Reply:
What is it in the JSON API?